README.md

```diff
@@ -1 +1,38 @@
-# hw01
\ No newline at end of file
+
+# My Autobiography
+
+
+My name is Tatiana Hernández and I am from Ohio. Currently, I am a graduate student in the MAPSS program. Some of my research interests include:
```

Comment:
Be careful of **unnecessary spacing** in your code. Whether or not you add these spaces, Markdown renders it the same way. This will be more important further into the course when we start to pay closer attention to coding style.